charts/bucketrepo/templates/_config.yaml.tpl

```
@@ -6,8 +6,10 @@ http:
 
 storage:
 {{- if .Values.config.storage.bucketUrl }}
+    enabled: true
     bucket_url: "{{ .Values.config.storage.bucketUrl }}"
 {{- else if and (hasKey .Values.jxRequirements "storage") ( .Values.jxRequirements.storage)  }}
+    enabled: true
 {{- range $key, $val := .Values.jxRequirements.storage }}
 {{- if eq "repository" $val.name }}
     bucket_url: "{{ $val.url }}"
```

As you wrote it storage could be enabled even if bucket_url isn't set.
Add say this would be the correct way:
```suggestion
{{- range $key, $val := .Values.jxRequirements.storage }}
{{- if eq "repository" $val.name }}
    enabled: true
    bucket_url: "{{ $val.url }}"
```